config/initializers/suppress_routing_error_logs.rb

```diff
@@ -0,0 +1,7 @@
+module NoRoutingErrorLogging
+  def log_error(env, wrapper)
+    super unless wrapper.exception.is_a? ActionController::RoutingError
```

Comment:
We might also want to make this dependent on the log level e.g. 
```
super unless wrapper.exception.is_a?(ActionController::RoutingError) && Rails.logger.level >=  Logger::Severity::ERROR
```

Reply:
@ulferts The log level in production is INFO (1) right now as far as I can tell. This would mean the routing errors would still be logged they way you suggest it. If we do want to make this dependent on the log level I would suggest to only log this in the DEBUG level.

Reply:
I would agree that we can enable this when set to `DEBUG` only